app/src/main/java/com/postpc/tenq/ui/activities/RoomActivity.java

```diff
@@ -25,7 +31,13 @@
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
+        // sets new action bar theme
         super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_room);
```

Comment:
Remove this line - it's being called with the binding root at line 42 already :)
Move line 39 to be below it and it should work fine